proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+* issue tracker (GitHub by default)
+* website 
+* release methodology and mechanics
+* social media accounts
+* community size and any existing financial sponsorship
+
+### Project Acceptance Process
+* Projects are required to present their proposal at a CPC meeting
+* Projects get accepted via a 2/3 supermajority vote of the CPC
+
+## III. Stages - Definitions & Expectations
+Every _$FOUNDATION_ project has an associated maturity level. Proposed _$FOUNDATION_ projects should state their preferred maturity level. Projects of all maturities have access to _$FOUNDATION_ resources.
+
+*note: all stage names are tbd pending outcome of [#44](https://github.com/nodejs/bootstrap/issues/44#issuecomment-440026298)*
+
+### Sandbox Stage
+**Definition** 
+Sandbox Stage projects are projects which the CPC believes are, or have the potential to be, important to the ecosystem of Top-Level Projects or the JS ecosystem as a whole. They may be early-stage projects just getting started, or they may be long-established projects with minimal resource needs. The Sandbox Stage provides a beneficial, neutral home for these projects in order to foster collaborative development and provide a path to deeper alignment with other _$FOUNDATION_ projects via the graduation process.
+
+**Examples**
+1. New projects that are designed to extend one or more _$FOUNDATION_ projects with functionality or interoperability libraries. 
+1. Independent projects that fit the _$FOUNDATION_ mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need).
+1. Projects commissioned or sanctioned by the _$FOUNDATION_.
+1. Any project that realistically intends to join the _$FOUNDATION_ Incubation or Top Level Stages in the future and wishes to lay the foundations for that transition.
+
+**Expectations**
+End users should evaluate Sandbox projects with care - some Sandbox projects may fail, and others may stay in the  Sandbox Stage indefinitely. Sandbox stage projects will receive minimal marketing support from the _$FOUNDATION_. Sandbox Stage projects will be reviewed on an annual basis; they may also request a status review by submitting a report to the CPC.
+
+**Acceptance Criteria**
+To be considered for Sandbox Stage, the project must meet the following requirements:
+* 2 CPC sponsors to champion the project & provide mentorship
+* A presentation to at the meeting of the CPC
+* Adherence to the _$FOUNDATION_ IP Policy
+* Upon acceptance, Sandbox projects must list their status prominently on website/readme
+
+### Incubating Stage*
+**Definition** 
+The Incubation Stage is for projects that are interested in reaching the Top-Level Stage, or want to have more access, participation, responsibilities and representation in the CPC. Incubating projects will receive mentorship from the CPC and are expected to actively develop their community of contributors, governance, and other project documentation. 
+
```

Comment:
I think we should add the following in respect to the comment above related to needing more detail as to what it means to be at this stage.

In order to support their active development, projects in the incubation stage have a higher level of access to marketing and other resources which will be agreed upon acceptance to the incubation stage and reviewed/agreed on a yearly basis.  Progress will be reviewed on a yearly basis and a project may be asked by the CPC to move back to the Sandbox stage if the level of development drops off or stalls.

Reply:
I'm OK with that. Adding for now as we refine what these stages are.